test/state/block.cpp

```diff
@@ -7,6 +7,9 @@
 
 namespace evmone::state
 {
+static constexpr auto GAS_LIMIT_ELASTICITY_MULTIPLIER = 2;
```

Comment:
Maybe move to the function body?

Reply:
I followed the style of this file - other constants are global